libcalico-go/lib/clientv3/felixconfig.go

```diff
@@ -113,4 +113,12 @@ func setDefaults(fc *apiv3.FelixConfiguration) {
 		disabled := apiv3.FloatingIPsDisabled
 		fc.Spec.FloatingIPs = &disabled
 	}
+	if fc.Spec.BPFConnectTimeLoadBalancing == nil {
```

Comment:
This should be the place where we also need to decide if the legacy `BPFConnectTimeLoadBalancingEnabled` is set, right? If it is is and none of the new ones are set, just set `BPFConnectTimeLoadBalancing=true` and `BPFHostNetworkedNATWithoutCTLB=disabled` and then you do not need to worry about `BPFConnectTimeLoadBalancingEnabled` in Felix. If the new one and the old one are set then we need to either raise a warning or fail validation or something, right? Once we deprecate the legacy option we would remove the code here.